src/bindings/python/tests/mock/ov_mock_py_frontend/CMakeLists.txt

```diff
@@ -4,18 +4,36 @@
 
 set(TARGET_FE_NAME "ov_mock_py_frontend")
 
-file(GLOB_RECURSE LIBRARY_SRC ${CMAKE_CURRENT_SOURCE_DIR}/*.cpp)
-file(GLOB_RECURSE LIBRARY_HEADERS ${CMAKE_CURRENT_SOURCE_DIR}/*.hpp)
+file(GLOB_RECURSE LIBRARY_SRC ${CMAKE_CURRENT_SOURCE_DIR}/src/*.cpp)
+file(GLOB_RECURSE LIBRARY_HEADERS ${CMAKE_CURRENT_SOURCE_DIR}/include/*.hpp)
 
 source_group("src" FILES ${LIBRARY_SRC})
 source_group("include" FILES ${LIBRARY_HEADERS})
 
-# Create shared library
-add_library(${TARGET_FE_NAME} SHARED ${LIBRARY_SRC} ${LIBRARY_HEADERS})
+add_library(${TARGET_FE_NAME} ${LIBRARY_SRC} ${LIBRARY_HEADERS})
 
-target_include_directories(${TARGET_FE_NAME} PRIVATE ${CMAKE_CURRENT_SOURCE_DIR})
+target_include_directories(${TARGET_FE_NAME}
+        PUBLIC ${CMAKE_CURRENT_SOURCE_DIR}/include
+        PRIVATE ${CMAKE_CURRENT_SOURCE_DIR}/src)
 
-target_link_libraries(${TARGET_FE_NAME} PRIVATE frontend_common::static)
+set(DEPENDENCIES frontend_common::static)
+
+if (ENABLE_OV_ONNX_FRONTEND)
+    list(APPEND DEPENDENCIES ov_onnx_frontend)
+    add_compile_definitions(ENABLE_OV_ONNX_FRONTEND)
+endif()
+
+if (ENABLE_OV_TF_FRONTEND)
+    list(APPEND DEPENDENCIES ov_tensorflow_frontend)
+    add_compile_definitions(ENABLE_OV_TF_FRONTEND)
+endif()
+
+if (ENABLE_OV_PADDLE_FRONTEND)
+    list(APPEND DEPENDENCIES ov_paddle_frontend)
```

Comment:
I suppose `ov_paddle_frontend` is named differently as a part of developer package.

Reply:
fixed